### ISSUE TITLE
Feature request: encourage 72-character lines

### ISSUE THREAD

**Issue by duncanjbrown** [jreybert/vimagit#140]
Thank you for this great plugin!

A small feature request.

In vimagit's commit mode, the first line of the commit is highlighted to encourage a 50-character length, but the rest of the commit has a line length of 80 characters. This means that my commit messages don't look quite right [on GitHub](https://github.com/blog/926-shiny-new-commit-styles) 🙂 . 

I have been manually setting `textwidth=72` in the vimagit buffer to make my messages conform. Could Magit set this?

If this is a simple change I'm happy to have a stab at a PR.

**Comment by jreybert** [comment]
Hi Duncan,
thanks for encouraging!

This is a good idea, but I am not sure I want to set `tw` for the user (I am still not sure...).

A first step will be a new user command: `VimagitLeaveCommit`. With this new command, adding following commands in your vimrc will set the textwidth when you are writing a commit message:

```
autocmd User VimagitEnterCommit setlocal textwidth=72                                                                                                                                                                                                                       
autocmd User VimagitLeaveCommit setlocal textwidth=0
```

Give a try on `next` branch, add User autocmd in your vimrc and tell me if it fits your needs.

**Comment by duncanjbrown** [comment]
Thank you. I've done as you suggest. Will live with it for a day or two and let you know.

**Comment by jreybert** [comment]
Merged in master and present in release 1.7.2

**Comment by duncanjbrown** [comment]
Can confirm that this is working well :+1:. Thanks!
